addons/partner_autocomplete/models/res_partner_autocomplete_sync.py

```diff
@@ -30,6 +33,7 @@ def start_sync(self):
                     _logger.warning('Send Partner to sync failed: %s', str(error))
 
             to_sync_item.write({'synched': True})
+        self.env['ir.cron']._notify_progress(len(to_sync_items))
```

Comment:
There is no commit in this cron, so it could simply
```py
return (len(to_sync_items), to_sync_count)
```
All this needs is to assign `action` in the cron action.